tests/init.js

```diff
@@ -22,6 +22,13 @@
  */
 'use strict';
 
+// Define global params needed for tests to run on existing app code
+var params = {};
+var search = {
+    'prefix' : '',
+    'state' : ''
+};
+
```

Comment:
NB This is the `tests/init.js` file, and these are dummy variables allowing the tests to run.

Reply:
@mossroy You're right that for tests we don't need to track variables in the same global way, because we don't have cancellation, but these variables are dummies that exist merely so that the code can run. with minimal variations. I can look to simplify. This was a kind of "patch" to get the tests to run on the same codebase...

Reply:
I have removed this from init.js. It now runs locally in tests.js.